src/libostree/ostree-repo-commit.c

```diff
@@ -2384,7 +2384,7 @@ get_modified_xattrs (OstreeRepo                       *self,
 
       if (label)
         {
-          GVariantBuilder *builder;
+          g_autoptr(GVariantBuilder) builder;
 
           /* ret_xattrs may be NULL */
           builder = ot_util_variant_builder_from_variant (ret_xattrs,
```

Comment:
Can you hoist up the initialization too?  I know it doesn't matter now, but I think autoptrs should *always* be initialized at a minimum as defense against code motion/refactoring.  I.e. if someone later adds code between the two that could throw we'd be freeing uninitialized memory.